tools/pages/admin/permit-holders/types.ts

```diff
@@ -7,6 +7,7 @@ import {
   Physician,
 } from '@lib/graphql/types';
 
+//Used in PermitHolderHeader and PersonalInformationCard components
```

Comment:
For these comments, also include information on what the type does, eg for this one you could do:
"Applicant data to show in Personal Information Card"
and that would be better than just having "used in x,y,z". We should replace the "Used in..." comments with what the type does